src/commits.js

```diff
@@ -184,17 +184,17 @@ function getIssueLink (match, id, remote, issueUrl) {
   return `${remote.url}/issues/${id}`
 }
 
-function getMergeLink (id, remote) {
+function getMergeLink (id, remote, options = {}) {
   if (!remote) {
     return null
   }
-  if (/bitbucket/.test(remote.hostname)) {
+  if ((/bitbucket/.test(remote.hostname) && options.platform === undefined) || options.platform === 'bitbucket') {
```

Comment:
Maybe we want to have configuration file with association between platform -> merge links?

Reply:
I plan on moving all platform logic to it's own area, which will include a solution for what this PR is trying to solve.

Reply:
Hello, any news of that ? thank you